### PR TITLE
WIP python-package-init.py: constraints

### PR DESCRIPTION
**Pull request by FRidh**
Builds upon https://github.com/costrouc/nixpkgs-dev-tools/pull/3

### CODE REVIEW

File: python/python-package-init.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i python3 -p python3 python36Packages.jinja2 python36Packages.setuptools
+#! nix-shell -i python3 -p python37 python37Packages.jinja2 python37Packages.setuptools
```
Comment (FRidh) [inline python/python-package-init.py:2]: Not actually using string interpolation so this change can be undone

**Comment by FRidh** [comment]
If going further with it, I think it's worth doing it in a separate repo, where we have a library because this is going to need unit tests.

**Comment by costrouc** [comment]
Agree. I am really glad that this script has been helpful and I would be happy to work on it further. Thinking of `nixpkgs-python-dev-tools` though not creative. If it makes it easier I would also like to add you as a `collaborator`. Thoughts?

**Comment by makefu** [comment]
This PR lingers a bit longer now, is it worth adding the functionality to nixpkgs-pytools by resolving the conflict or should we just close this PR here?

**Comment by makefu** [comment]
closing for now, thanks for the contribution anyway :)

**Pull request closed without merging**